interfaces/builtin/serial_port.go

```diff
@@ -72,7 +72,8 @@ func (iface *serialPortInterface) String() string {
 //  - ttySX (UART serial ports)
 //  - ttyOX (UART serial ports on ARM)
 //  - ttymxcX (serial ports on i.mx6UL)
```

Comment:
Missed adding ttySCX to the comment. I can do that in a follow-up (I noticed AMA is missing too).